docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -236,6 +241,37 @@ file paths for 2019 forecast data:
 
 see :ref:`stomach cancer model <2017_stomach_cancer>`
 
+Data Sources Table
+
++----------------------+----------------------------+-----------------------------------+-------------------------------------------+
+| Input parameter      | Value                      | Note                              | Reference                                 |
++----------------------+----------------------------+-----------------------------------+-------------------------------------------+
+| Mean sojourn time    |  2.37 years                | The UI from Bae is actually       | Bae J Prev Med Pub Health 2014            |
+|                      |  (95%CI: 1.78 to 2.96)     | (95%CI: 1.92 to 2.96) which is    |                                           | 
+|                      |                            | skewed. To make a symmetrical     |                                           |      
+|                      |                            | normal distribution, we make the  |                                           | 
+|                      |                            | lower bound shorter to 1.78       |                                           |
++----------------------+----------------------------+-----------------------------------+-------------------------------------------+
+
+
+Full references
+
+  - Bae J-M, Shin SY, Kim EH. Mean Sojourn Time of Preclinical Gastric Cancer in Korean Men: A Retrospective Observational Study. J Prev Med Pub Health 2014; 47: 201–5.
+
+Validation and verification
+
+  - i_pc should validate to ~ i_c414/(1-prev_c414) shifted by the 2.37 years forward. i_c414 and prev_c414 comes from forecast.
+  - i_c should validate to ~ 1/MST
+  - prevalence of pc ~ i_pc/(1-prev_c414) X 2.37 shifted by the 2.37 years forward. 
+  - prevalence of C should be 0 at the start and eventually catch up to forecast prevalence in later years
```

Comment:
"Later years" meaning sooner than MST + 10

Reply:
For prevalence of PC state, do we need shift it by mean sojourn time forward if you apply `i_pc` directly from Sim?

Reply:
@yongqx yes , you are right, the prevalence of PC doesn't need to be shifted!